packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -61,6 +61,24 @@ export class MetamaskPage implements WalletPage {
     });
   }
 
+  async cancelAllTxInQueue() {
```

Comment:
generally if you will look into `this.navigate() `- it do same thing that you want except Close all extension pages(). Why pages closing regards to this function?

Reply:
because when we run the suite with transaction tests, we don't visit the wallet page between tests, and TXs are not canceled. 
using the `navigate` function in `beforeEach` on purpose would be wrong, because it would be unclear why we are doing this.

that's why I wrote a similar function, whose only purpose is to close all TXs and unnecessary wallet windows (because open wallet windows also break the tests)